index.js

```diff
@@ -15,7 +15,7 @@ gesture.drag.configureHoldPulse({
 	events: [{name: 'hold', time: 400}],
 	resume: false,
 	moveTolerance: 16,
```

Comment:
If we're using `onLeave`, can we remove the explicit declaration of the `moveTolerance` property here?